src/Raven.Server/Documents/ChangesClientConnection.cs

```diff
@@ -574,7 +583,64 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ms))
                             break;
 
                         ms.TryGetBuffer(out ArraySegment<byte> bytes);
-                        await _webSocket.SendAsync(bytes, WebSocketMessageType.Text, true, _disposeToken);
+
+                        var sendTask = _webSocket.SendAsync(bytes, WebSocketMessageType.Text, true, _disposeToken);
+                        var sendSp = Stopwatch.StartNew();
+
+                        while (true)
+                        {
+                            var waitTask = TimeoutManager.WaitFor(TimeSpan.FromSeconds(5), _disposeToken);
+
+                            var waitResult = Task.WaitAny(sendTask, waitTask);
```

Comment:
This should be `await Task.WhenAny()`, no?

Reply:
Yes. Done.